install.sh

```diff
@@ -102,6 +102,21 @@ deploy "${BUILDDIR}/manifests/sysmgmt.yaml"
 # Deploy Nexus
 deploy "${BUILDDIR}/manifests/nexus.yaml"
 
+# Deploy Vshasta specific services
+function is_vshasta_node {
+    # This is the best check for an image specifically booted to vshasta
+    [[ -f /etc/google_system ]] && return 0
+
+    # metal images can still be booted on GCP, so check if there are any disks vendored by Google
+    # if not, we conclude that this is not GCP
+    lsblk --noheadings -o vendor | grep -q Google
+    return $?
+}
+
+if is_vshasta_node; then
+    deploy "${BUILDDIR}/manifests/vshasta.yaml"
```

Comment:
Since in vshasta the `install.sh` is called only from the PIT node where `/etc/google_system` exists, this isn't a blocking issue. 

I agree at some point we should look into a better check for vshasta. Right now I'm copying the existing behavior in csm-testing: https://github.com/Cray-HPE/csm-testing/blob/main/goss-testing/automated/run-ncn-tests.sh#L48-L56 
So if we make a change to one of these checks we should ensure its present in the other location. This also means that if the Goss tests are ran on a non-PIT node it will run all of the tests that should be skipped for vshasta. I'll open a ticket for this

Reply:
Eric, do you approve or should this wait?

Reply:
I approve.  Resolving the conversation now.